docs/model_server_rest_api.md

```diff
@@ -147,7 +147,7 @@ Read more about *Get Model Metadata API* usage [here](./../example_client/README
 ## Predict API <a name="predict"></a>
 * Description
 
-Sends requests via TensorFlow Serving RESTful API using images in numpy format. It displays performance statistics and optionally the model accuracy.
+Sends requests via TensorFlow Serving RESTful API using images in base64 encoded string or numpy format. It displays performance statistics and optionally the model accuracy.
```

Comment:
I would put it as numpy array or binary format.
base64 is just encoding to json but originally the input data is an array or binary image.